### PR TITLE
en: replace Mydumper with Dumpling

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

Replace Mydumper with Dumpling in backup docs.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/562
- Other reference link(s):

### CODE REVIEW

File: en/backup-to-gcs.md [hunk 1]
```diff
@@ -134,30 +141,71 @@ After creating the `Backup` CR, you can use the following command to check the b
...
+
+    ```
+    tableFilter:
+    - "*.*"
+    - "!db.table"
     ```
 
 * `.spec.storageClassName`: the persistent volume (PV) type specified for the backup operation. If this item is not specified, the value of the `default-backup-storage-class-name` parameter is used by default. This parameter is specified when TiDB Operator is started, and is set to `standard` by default.
-* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than the size of the TiDB cluster to be backed up.
+* `.spec.storageSize`: the PV size specified for the backup operation (`100 Gi` by default). This value must be greater than the size of the TiDB cluster to be backed up.
```
Comment (DanielZhangQD) [inline en/backup-to-gcs.md:204]:
```suggestion
* `.spec.storageSize`: the PV size specified for the backup operation (`100 Gi` by default). This value must be greater than the data size of the TiDB cluster to be backed up.
```

File: en/backup-to-gcs.md [hunk 2]
```diff
@@ -134,30 +141,71 @@ After creating the `Backup` CR, you can use the following command to check the b
...
+
+* `.spec.cleanPolicy`: The clean policy of the backup file when the backup CR is deleted after the backup is completed.
+
+    Three clean policies are supported:
+
+    * `Retain`: On any circumstances, retain the backup file when deleting the backup CR.
+    * `Delete`: On any circumstances, delete the backup file when deleting the backup CR.
+    * `OnFailure`: If the backup fails, delete the backup file when deleting the backup CR.
+
+    If this field is not configured, or if you configure a value other than the three policies above, the backup file is retained.
```
Comment (DanielZhangQD) [inline en/backup-to-gcs.md:172]: s/file/data

File: en/backup-to-gcs.md [hunk 3]
```diff
@@ -134,30 +141,71 @@ After creating the `Backup` CR, you can use the following command to check the b
...
+
+    In extreme cases, if TiDB Operator fails to access the database, TiDB Operator cannot automatically recover the value of `tikv_gc_lifetime` and treats the backup as failed. At this time, you can view `tikv_gc_lifetime` of the current TiDB cluster using the following statement:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    If the value of `tikv_gc_lifetime` is set too large (usually 10m), you need to set it back to the previous value.
```
Comment (DanielZhangQD) [inline en/backup-to-gcs.md:162]: 这里直接把那个命令拿过来吧，不用连到 dumpling 那个文档了

File: en/backup-to-aws-s3-using-br.md [hunk 4]
```diff
@@ -323,14 +333,43 @@ After creating the `Backup` CR, use the following command to check the backup st
...
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc_life_time";
+    ```
+
+    If the value of `tikv_gc_lifetime` is set too large (usually 10m), you need to set it back to the previous value.
+
+- `.spec.cleanPolicy`: The clean policy of the backup file when the backup CR is deleted after the backup is completed.
```
Comment (lichunzhu) [inline en/backup-to-aws-s3-using-br.md:356]: No need of `after the backup is completed.`

File: en/backup-to-gcs.md (hunk 3 above)
Comment (ran-huang) [reply en/backup-to-gcs.md]: Updated.

File: en/backup-to-gcs.md (hunk 2 above)
Comment (ran-huang) [reply en/backup-to-gcs.md]: Updated.

File: en/backup-to-aws-s3-using-br.md (hunk 4 above)
Comment (ran-huang) [reply en/backup-to-aws-s3-using-br.md]: Updated.

**Pull request merged**